<html><body><h1>..\P31\.\tests\test_Problem31_GeneratedSolution.py</h1>Killed 4 out of 13 mutants<h2>Survived</h2>Survived mutation testing. These mutants show holes in your test suite.<h3>Mutant 116</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -4,7 +4,7 @@
 import Problem31_GeneratedSolution as module_0
 
 
-@pytest.mark.xfail(strict=True)
+@pytest.mark.xfail(strict=False)
 def test_case_0():
     int_0 = 1140
     module_0.findOrder(int_0, int_0)
</pre><h3>Mutant 118</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -6,7 +6,7 @@
 
 @pytest.mark.xfail(strict=True)
 def test_case_0():
-    int_0 = 1140
+    int_0 = 1141
     module_0.findOrder(int_0, int_0)
 
 
</pre><h3>Mutant 119</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -6,7 +6,7 @@
 
 @pytest.mark.xfail(strict=True)
 def test_case_0():
-    int_0 = 1140
+    int_0 = None
     module_0.findOrder(int_0, int_0)
 
 
</pre><h3>Mutant 120</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -11,7 +11,7 @@
 
 
 def test_case_1():
-    bool_0 = True
+    bool_0 = False
     list_0 = []
     var_0 = module_0.findOrder(bool_0, list_0)
 
</pre><h3>Mutant 123</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -13,7 +13,7 @@
 def test_case_1():
     bool_0 = True
     list_0 = []
-    var_0 = module_0.findOrder(bool_0, list_0)
+    var_0 = None
 
 
 @pytest.mark.xfail(strict=True)
</pre><h3>Mutant 124</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -16,7 +16,7 @@
     var_0 = module_0.findOrder(bool_0, list_0)
 
 
-@pytest.mark.xfail(strict=True)
+@pytest.mark.xfail(strict=False)
 def test_case_2():
     int_0 = 1140
     set_0 = {int_0, int_0, int_0, int_0}
</pre><h3>Mutant 126</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -18,7 +18,7 @@
 
 @pytest.mark.xfail(strict=True)
 def test_case_2():
-    int_0 = 1140
+    int_0 = 1141
     set_0 = {int_0, int_0, int_0, int_0}
     module_0.findOrder(int_0, set_0)
 
</pre><h3>Mutant 127</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -18,7 +18,7 @@
 
 @pytest.mark.xfail(strict=True)
 def test_case_2():
-    int_0 = 1140
+    int_0 = None
     set_0 = {int_0, int_0, int_0, int_0}
     module_0.findOrder(int_0, set_0)
 
</pre><h3>Mutant 128</h3><pre>--- ..\P31\.\tests\test_Problem31_GeneratedSolution.py
+++ ..\P31\.\tests\test_Problem31_GeneratedSolution.py
@@ -19,6 +19,6 @@
 @pytest.mark.xfail(strict=True)
 def test_case_2():
     int_0 = 1140
-    set_0 = {int_0, int_0, int_0, int_0}
+    set_0 = None
     module_0.findOrder(int_0, set_0)
 
</pre></body></html>